plugin/fuzz/source/dextool/plugin/fuzz/backend/fuzzvariant.d

```diff
@@ -626,45 +627,52 @@ import cpptooling.data.type;
 			    string min = minmax["min"];
 			    string max = minmax["max"];
 			    string type_type = minmax["type"];
-                string type_ns = minmax["namespace"];
+			    string type_ns = minmax["namespace"];
 			    
 			    final switch (type_type) {
 			    case "SubType":
-				stmt(E(ciface.name.toLower ~ "." ~ ditem.name) = E(
-									   `randomGenerator->generate("` ~ type ~ ` ` ~ ciface.name
-									   ~ ` ` ~ ditem.name ~ `", `~min~`, `~max~`)`));
+				string var = format("%s.%s", ciface.name.toLower, ditem.name);
+				string expr = format("randomGenerator->generate(\"%s %s %s\", %s, %s)",
```

Comment:
The backtick, `, can also be used for strings. Because it is uncommon in "other" languages it means it almost always "just works".